jsonschema/cli.py

```diff
@@ -252,7 +262,34 @@ def load(_):
                 raise _CannotLoadFile()
         instances = ["<stdin>"]
 
-    validator = arguments["validator"](schema)
+    if arguments["base_uri"] is None:
+        resolver = None
+    else:
+        scheme = urllib.parse.urlsplit(arguments["base_uri"]).scheme
```

Comment:
There's way too much code here. What goes wrong when this whole else block disappears? (And when the value as given on the command line is used exactly as-is?)